src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -228,11 +228,11 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
         example: 15
         values: [max, 1..1000000]
 
-      "randomSeed" required if tiebreakMode is "Random", "Previous round counts (then random)", or "Generate permutation"
+      "randomSeed" required if tiebreakMode is "random", "previousRoundCountsThenRandom", or "generatePermutation"
         the integer seed for the application's pseudorandom number generator
         value: [-140737488355328..140737488355327]
 
-      "multiSeatBottomsUpPercentageThreshold" required if winnerElectionMode is "Bottoms-up using percentage threshold" and numberOfWinners is 0
+      "multiSeatBottomsUpPercentageThreshold" required if winnerElectionMode is "bottomsUpUsingPercentageThreshold" and numberOfWinners is 0
```

Comment:
Can you please Ctrl-Shift-F through the entire project (including comments and this .txt file; with the exception of the hint .txt files since those should show the GUI labels) and replace all instances of every single enum value you're modifying in this PR? Looks like you missed at least one instance of "Bottoms-up using percentage threshold" here.

Reply:
Hmm. I already tried to search through everything and update it. I'm confused by your comment -- I _did_ catch this and update it, didn't I?

Reply:
There's one below it, but GH acts dumb if I comment on lines outside the diff.